File: runtime-language-decision/benchmarks/cpu.py
import time,json,statistics
xs=[];z=0
for _ in range(5):
 s=0;t=time.perf_counter()
 for i in range(10_000_000): s=(s+((i*31)%97)) & 0xffffffff
 xs.append((time.perf_counter()-t)*1000);z^=s
print(json.dumps({'language':'python','cpu_loop_ms':statistics.median(xs),'guard':z}))

File: runtime-language-decision/benchmarks/cpu.go
package main
import("encoding/json";"fmt";"sort";"time")
func main(){xs:=make([]float64,5);z:=0;for r:=0;r<5;r++{s:=0;t:=time.Now();for i:=0;i<10000000;i++{s=(s+((i*31)%97))&0xffffffff};xs[r]=float64(time.Since(t).Microseconds())/1000;z^=s};sort.Float64s(xs);b,_:=json.Marshal(map[string]any{"language":"go","cpu_loop_ms":xs[2],"guard":z});fmt.Println(string(b))}
